Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -25,7 +25,8 @@ class BrowserViewController: UIViewController {
     private var webViewContainer: UIView!
     private let uriFixup = URIFixup()
     private var screenshotHelper: ScreenshotHelper!
-    private var homePanelIsInline: Bool = false
+    private var homePanelIsInline = false
+    private var searchLoader: SearchLoader!
```

Comment:
What would you propose -- creating a class variable that we set when opening the SVC, then nilling it out when we close it? We could, I guess, but that just adds some extra housekeeping and complexity. `SearchLoader` doesn't have any significant memory footprint.